docs/concepts/ipld.md

```diff
@@ -11,4 +11,53 @@ related:
 
 # The InterPlanetary Linked Data (IPLD) model
 
-<ContentStatus />
+The Interplanetary Linked Data (IPLD) model is a single namespace for all hash-inspired protocols. This allows for interoperable data formats. For example, you can reference a Git commit to a Bitcoin transaction.
+
+As mentioned in the [specifications](https://github.com/ipld/specs), the goal of IPLD is to enable decentralized data-structures that are universally addressable and linkable, which in turn will enable more decentralized applications. These data structures allow us to do for data what URLs and links did for HTML web pages.
+
+IPFS, Bitcoin, and other applications use hash-linked data structures ([Merkle trees](https://docs-beta.ipfs.io/concepts/merkle-dag/)) to verify data integrity. However, each protocol has a very different format. IPLD's goal is to provide a data layer definition that allows the linking of different content-addressed data types.
+
+## Where does IPLD fit in?
+
+IPLD acts as a layer between networking protocols and higher-level applications. This enables developers to build decentralized applications without having to worry about lower-level changes. Both upper- and lower-layer technologies can grow independently, while IPLD acts as a middle layer that resolves the two. This is known as "thin waist" or "narrow waist" protocol.
```

Comment:
Is there a way to rephase this a little so it doesn't necessarily have to be "between" networking and higher level applications?  It's also possible to use IPLD entirely without a network in the picture.

Kind of similar to git: would you say git is "between" networking and the application of version control?  Not really, right?  Git means you don't have to *care* about networking at all when defining any of the version control semantics -- `git commit` is an entirely local operation; networking doesn't really show up anywhere outside of `git fetch`.  IPLD is similar: making an IPLD object and getting a CID for it is an entirely local operation.

I'm still fine with this paragraph overall.  It's true that we still refer to it as the "thin waist" in a lot of ways.  Just thinking outloud about other more inclusive ways we could try to phrase it in the future.

Reply:
Good point. I'd be open to any suggestions